lxd/api_project.go

```diff
@@ -1457,6 +1457,38 @@ func projectValidateConfig(s *state.State, config map[string]string, projectName
 		return fmt.Errorf("Failed loading storage pool names: %w", err)
 	}
 
+	// Per-network project limits for uplink IPs only make sense for projects with their own networks.
+	if shared.IsTrue(config["features.networks"]) {
+		// Get networks that are allowed to be used as uplinks by this project.
+		allowedUplinkNetworks, err := network.AllowedUplinkNetworks(s, config)
+		if err != nil {
+			return nil
+		}
+
+		// Add network-specific config keys.
+		for _, networkName := range allowedUplinkNetworks {
+			// lxdmeta:generate(entities=project; group=limits; key=limits.networks.uplink_ips.ipv4.NETWORK_NAME)
+			// This represents the maximum value for IPV4 addresses available on a network
```

Comment:
I think this can be clarified a bit, how about

> Maximum number of IPv4 addresses that this project can consume from the specified uplink network. These IPs can be consumed by networks, forwards and load-balancers.

as "maximum value" is a bit vague IMHO.